packit/config/aliases.py

```diff
@@ -113,6 +122,24 @@ def get_build_targets(*name: str, default=DEFAULT_VERSION) -> Set[str]:
     return possible_sys_and_versions
 
 
+def get_valid_build_targets(*name: str, default: str = DEFAULT_VERSION) -> set:
+    """
+    Function generates set which contains build targets available also in copr chroots.
+
+    :param name: name(s) of the system and version or target name. (passed to
+                packit.config.aliases.get_build_targets() function)
+            or target name (e.g. "fedora-30-x86_64"/"fedora-stable-x86_64")
+    :param default: used if no positional argument was given
+    :return: set of build targets available also in copr chroots
+    """
+    build_targets = get_build_targets(*name, default)
+    logger.info(f"Build targets: {build_targets} ")
+    copr_chroots = CoprHelper.get_available_chroots()
+    logger.info(f"Copr chroots: {copr_chroots} ")
+    logger.info(f"Result set: {set(build_targets) & set(copr_chroots)}")
+    return set(build_targets) & set(copr_chroots)
```

Comment:
@sakalosj It probably does not work from what we are seeing [in test PR](https://github.com/packit/hello-world/pull/130#issuecomment-723756885). It looks like `get_available_chroots` does not return valid values.

It adds the `fedora-34` but it actually is not present:
![Screenshot from 2020-11-09 08-36-41](https://user-images.githubusercontent.com/20214043/98512626-e477d800-2266-11eb-9c72-b18ba9093fcf.png)

Reply:
this behaves correctly 

```python
>>>from packit.config.aliases import get_valid_build_targets
>>>get_valid_build_targets()
{'fedora-33-x86_64', 'fedora-31-x86_64', 'fedora-32-x86_64'}
>>>
```

will look into packit-service